src/components/custom-list-item.js

```diff
@@ -115,7 +115,7 @@ export const CustomListItem = ({
             button
             key={item.name}
             selected={selected}
-            onClick={() => handleItemClicked(item)}
+            onMouseDown={() => handleItemClicked(item)}
```

Comment:
Why this change ? is this related to filter extension ?

Reply:
This is a correction of an existing bug in the master, when we delete an existing filter ... it tries to get the filter again, after it's deletion ...